src/lua/utils.h

```diff
@@ -71,13 +72,24 @@ extern struct lua_State *tarantool_L;
 extern uint32_t CTID_CHAR_PTR;
 extern uint32_t CTID_CONST_CHAR_PTR;
 extern uint32_t CTID_UUID;
+extern uint32_t CTID_DATETIME;
+extern uint32_t CTID_INTERVAL;
```

Comment:
This id is not defined anywhere, and the intervals are a part of one of the next stages AFAIR.

Reply:
Correct. It is not there already, in the branch. (Fixed)